hive/agents/rainbow.py

```diff
@@ -18,17 +18,17 @@
 from hive.replays.replay_buffer import BaseReplayBuffer
 from hive.utils.loggers import Logger
 from hive.utils.schedule import Schedule
-from hive.utils.utils import LossFn, OptimizerFn, seeder
+from hive.utils.utils import LossFn, OptimizerFn
 
 
 class RainbowDQNAgent(DQNAgent):
     """An agent implementing the Rainbow algorithm."""
 
```

Comment:
I believe the `legal_moves_rainbow` agent does not need any update. But could you also confirm that?

Reply:
Nope, should be good as is